api/Helpers/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -47,7 +47,8 @@ public static bool IsIdirUser(this ClaimsPrincipal claimsPrincipal)
 
         public static bool IsVcUser(this ClaimsPrincipal claimsPrincipal)
             => claimsPrincipal.HasClaim(c => c.Type == CustomClaimTypes.PreferredUsername) &&
-               claimsPrincipal.FindFirstValue(CustomClaimTypes.PreferredUsername).EndsWith("@vc");
+               (claimsPrincipal.FindFirstValue(CustomClaimTypes.PreferredUsername).EndsWith("@accredited-lawyer-bcpc") ||
+                claimsPrincipal.FindFirstValue(CustomClaimTypes.PreferredUsername).EndsWith("@vc"));
```

Comment:
Would it make sense to have possible suffixes as a configuration, to account for potential changes in the future?